tasks/terraform-custom-image.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Should this task be removed? I'm guessing it was replaced by `tasks/packer-custom-image.yml`?